### PR TITLE
docs: keyring HTTP API documentation

### PR DESCRIPTION
**Pull request by tgross**
Preview: https://nomad-8bi9ilj87-hashicorp.vercel.app/api-docs/operator/keyring

### CODE REVIEW

File: website/content/api-docs/operator/keyring.mdx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+---
+
+# Keyring Operator HTTP API
+
+The `/operator/keyring` endpoints manage encryption root keys used for storing
+variables and signing workload, including examining active encryption keys,
+rotating keys, or removing unused keys.
+
+See the [Key Management] documentation for information how these capabilities
+are used. For a CLI to perform these operations manually, please see the
```
Comment (DerekStrickland) [inline website/content/api-docs/operator/keyring.mdx:15]:
```suggestion
are used. For instructions on how to use the CLI to perform these operations manually, please see the
```

This is a suggestion only. What you wrote it grammatically correct, it just didn't feel  natural to me.  This is just subjective feedback so feel free to ignore.

**Review by DerekStrickland — APPROVED** [review]
LGTM.  One non-blocking suggestion.

File: website/content/api-docs/operator/keyring.mdx (hunk 1 above)
Comment (tgross) [reply website/content/api-docs/operator/keyring.mdx]: Sure, that works

File: website/content/api-docs/operator/keyring.mdx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+layout: api
+page_title: Keyring - Operator - HTTP API
+description: |-
+  The /operator/keyring endpoints provide tools for Nomad operators to interact with the root keyring.
+---
+
+# Keyring Operator HTTP API
+
+The `/operator/keyring` endpoints manage encryption root keys used for storing
+variables and signing workload, including examining active encryption keys,
```
Comment (angrycub) [inline website/content/api-docs/operator/keyring.mdx:11]:
```suggestion
variables and signing workload identities, including examining active encryption keys,
```

**Review by angrycub — APPROVED** [review]
One suggestion that is totally optional and LGTM.

**Pull request merged**